### PR TITLE
Minutes from Wednesday

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

File: SR2020/2020-07-08.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Andy Barrett-Sprot
+- Andy Busse
+- Antoine Petty
+- Peter Law
+
+## Topics
+
+### Released 0.3.2
+
+This relaese has lots of fixes.
```
Comment (Adimote) [inline SR2020/2020-07-08.md:16]: [nit] typo
```suggestion
This release has lots of fixes.
```

File: SR2020/2020-07-08.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+### Preparation for the competition
+
+We’ll have another practise competition on Friday afternoon. Aim for a full
+dress rehearsal.
+
+### How to run the simulation
+
+In order to ensure consistent behaviour between runs, we’re aiming to run the
+simulation on one machine.
```
Comment (Adimote) [inline SR2020/2020-07-08.md:57]: (per weekend)

File: SR2020/2020-07-08.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2020/2020-07-08.md:16]: Ah yes, the daemon relaese.

File: SR2020/2020-07-08.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2020/2020-07-08.md:57]:
```suggestion
simulation on one machine per weekend.
```

**Pull request merged**